plugins/LilyPad/InputManager.cpp

```diff
@@ -433,7 +433,6 @@ Device *InputDeviceManager::GetActiveDevice(InitInfo *info, unsigned int *uid, i
 							  (devices[i]->oldVirtualControlState[j] > 31*FULLY_DOWN/32 && devices[i]->virtualControlState[j] < 7*FULLY_DOWN/8))) {
 									continue;
 						}
-						devices[i]->virtualControls[j].uid = devices[i]->virtualControls[j].uid;
```

Comment:
Are you the line is useless. It could be mistype of a wrong variable. At work I got a similar bug, I use the wrong variable on the right. Dunno if there any info on git history